next.config.mjs

```diff
@@ -23,6 +23,36 @@ const nextConfig = {
         port: '',
         pathname: '/a/**',
       },
+      {
+        protocol: 'https',
+        hostname: 'raw.githubusercontent.com',
```

Comment:
Might lead to a security issue if are not strict enough (mainly used to get icons from angleprotocol, sushiswap and trustwallet)

Reply:
We must be stricted.